utils/generator-utils.stanza

```diff
@@ -209,14 +209,16 @@ defn connected-group (obj: Pin|Net) :
 ; =======================================
 ; Attach the DNP (do not populate) to objects (components, array of instances, or module)
 ; =======================================
-public defn do-not-populate (objs:Seqable<JITXObject>) :
+public defn dnp (objs:Seqable<JITXObject>) :
```

Comment:
You could but I think this would actually make the code more complicated. You would need to add another branch that handles the `JITXObject`. I think with this approach you can reuse the existing code.

Reply:
The issue is the `for obj in objs`?

Reply:
yes - you would need to differentiate between the seqable and the object instance.